website/docs/reference/widgets/multiselect.md

```diff
@@ -14,46 +14,58 @@ These properties are customizable options present in the property pane of the wi
 
 ### Data
 
-#### Options `array`
+#### Source Data `array<object>`
 
 <dd>
 
-Use to set options in the list of the MultiSelect widget. Options must be specified as an array of objects with a `label` and `value` property. The `label` property represents the text that's displayed to the user, while the `value` property is the actual data that's stored and used in your application. For example:
+Use to set source data of the MultiSelect widget. Source Data must be specified as an array of objects. For example:
 
 
 ```js
 [
   {
-    "label": "Blue",
-    "value": "BLUE"
+    name: "Blue",
+    code: "BLUE",
   },
   {
-    "label": "Green",
-    "value": "GREEN"
+    name: "Green",
+    code: "GREEN",
   },
   {
-    "label": "Red",
-    "value": "RED"
-  }
-]
+    name: "Red",
+    code: "RED",
+  },
+];
 ```
 
-You can dynamically generate options by fetching data from queries or JS functions by binding the response to the **Options** property. For example, if you have a query named `fetchData`, you can bind its response using:
+You can dynamically generate source data by fetching data from queries or JS functions and binding the response to the **Source Data** property. For example, if you have a query named `fetchData`, you can bind its response using:
```

Comment:
```suggestion
You can dynamically generate options by fetching data from queries or JS functions and binding the response to the **Source Data** property. For example, if you have a query named `fetchData`, you can bind its response using:
```

Reply:
You also need to tell them they have to specify the label and value properties when they use query/JS

Reply:
@Pranay105 the above change is not done. "You also need to tell them they have to specify the label and value properties when they bind using JS"